modules/core/src/main/java/org/apache/ignite/mxbean/CacheMetricsMXBean.java

```diff
@@ -343,4 +343,12 @@
      */
     @MXBeanDescription("Disable statistic collection for the cache.")
     public void disableStatistics();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("True if index rebuilding in progress.")
+    @Override public boolean isIndexRebuildInProgress();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Number of keys processed during index rebuilding.")
```

Comment:
Please add clear explanation how to interpret result of this method to JMX description as well.